x-pack/plugins/kubernetes_security/common/translations.ts

```diff
@@ -58,3 +58,17 @@ export const COUNT_WIDGET_CONTAINER_IMAGES = i18n.translate(
     defaultMessage: 'Container Images',
   }
 );
+
+export const CONTAINER_NAME_SESSION = i18n.translate(
+  'xpack.kubernetesSecurity.containerNameWidget.ContainerImage',
```

Comment:
should be camelCase => `xpack.kubernetesSecurity.containerNameWidget.containerImage`